server/tests/api/api_stats_test.py

```diff
@@ -236,12 +236,10 @@ def side_effect(query, places, sv_only):
             assert result == expected_sv_only_result
 
 
-class TestGetStatVarGroup(unittest.TestCase):
```

Comment:
to keep things consistent, maybe we can extract these tests out into an "api_variable_test.py" and "api_variable_group_test.py" files?

Reply:
done